src/agent/multitype_queue/include/multitype_queue.hpp

```diff
@@ -44,10 +44,11 @@ class MultiTypeQueue : public IMultiTypeQueue
     /**
      * @brief Constructor.
      *
+     * @param dbFolderPath The path to the database folder.
      * @param size The maximum number of items in the queue.
      * @param timeout The timeout period in seconds.
      */
-    MultiTypeQueue(size_t size = DEFAULT_MAX, int timeout = DEFAULT_TIMEOUT_S);
+    MultiTypeQueue(const std::string& dbFolderPath, size_t size = DEFAULT_MAX, int timeout = DEFAULT_TIMEOUT_S);
```

Comment:
Don't we need a default value for the DB folder path?

Reply:
Same for the command store DB.

Reply:
@TomasTurina We don't need that as we always need to define one at code level. The configuration parser will provide a default value, but neither the Queue nor the Command Store need to apply a default value.